src/etc/module.xml

```diff
@@ -6,6 +6,7 @@
             <module name="Magento_InventorySales"/>
             <module name="Magento_InventorySalesApi"/>
             <module name="Magento_InventoryShipping"/>
+            <module name="Magento_InventoryShippingAdminUi"/>
```

Comment:
I'm not sure if we should really add that dependency. Technically, we do not depend on it. We might disable an observer from it or remove a layout block from it, but this won't break if the module is not present. Hence, I think there should not be a dependency, so that people can theoretically still remove this module.